src/Catalyst.Core.Modules.Kvm/DeltaExecutor.cs

```diff
@@ -298,20 +253,15 @@ private void Execute(ContractEntry entry, StateUpdate stateUpdate, ITxTracer txT
                 if (substate.ShouldRevert || substate.IsError)
                 {
                     if (_logger.IsEnabled(LogEventLevel.Verbose))
-                    {
                         _logger.Verbose("Restoring state from before transaction");
-                    }
-                    
+
                     _stateProvider.Restore(stateSnapshot);
                     _storageProvider.Restore(storageSnapshot);
                 }
                 else
                 {
-                    if (entry.IsValidDeploymentEntry)
-                    {
-                        DeployCode(env, substate, ref unspentGas, spec);
-                    }
-                    
+                    if (entry.IsValidDeploymentEntry) DeployCode(env, substate, ref unspentGas, spec);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4600656)